style/index.css

```diff
@@ -1 +1,222 @@
-/* Add CSS styling here */
\ No newline at end of file
+/* http://meyerweb.com/eric/tools/css/reset/
+v2.0 | 20110126
+License: none (public domain)
+*/
+
+html, body, div, span, applet, object, iframe,
+h1, h2, h3, h4, h5, h6, p, blockquote, pre,
+a, abbr, acronym, address, big, cite, code,
+del, dfn, em, img, ins, kbd, q, s, samp,
+small, strike, strong, sub, sup, tt, var,
+b, u, i, center,
+dl, dt, dd, ol, ul, li,
+fieldset, form, label, legend,
+table, caption, tbody, tfoot, thead, tr, th, td,
+article, aside, canvas, details, embed,
+figure, figcaption, footer, header, hgroup,
+menu, nav, output, ruby, section, summary,
+time, mark, audio, video {
+    margin: 0;
+	padding: 0;
+	border: 0;
+    /* border: 1px solid grey; */
+	font-size: 100%;
+	font: inherit;
+	vertical-align: baseline;
+}
+/* HTML5 display-role reset for older browsers */
+article, aside, details, figcaption, figure,
+footer, header, hgroup, menu, nav, section {
+	display: block;
+}
+body {
+	line-height: 1;
+}
+ol, ul {
+	list-style: none;
+}
+blockquote, q {
+	quotes: none;
+}
+blockquote:before, blockquote:after,
+q:before, q:after {
+	content: '';
+	content: none;
+}
+table {
+	border-collapse: collapse;
+	border-spacing: 0;
+}
+
+/* End CSS Reset */
+
+/* General Styles */
+
+* {
+    max-width: 100%;
+}
+
+html {
+    font-size: 62.5%;
+}
+
+body {
+    font-size: 1.6rem;
+    line-height: 1.5;
+    font-family: 'Montserrat', sans-serif;
+}
+
+h1, h2, h3, h4, h5, h6 {
+    font-size: 2.5rem;
+    font-family: 'Poppins', sans-serif;
+    letter-spacing: 3px;
+}
+
+a {
+    text-decoration: none;
+}
+
+/* Header */
+
+header {
+    Background: #026670;
+    height: 10vh;
+    padding: 2%;
+    display: flex;
+    flex-direction: column;
+    justify-content: space-around;
+    /* margin: 2% 0; */
+}
+
+nav {
+    margin: 1%;
+}
+
+nav a {
+    background: #EDEAE5;
+    padding: 2px;
+    color: black;
+    /* border: 1px solid #9FEDD7; */
+}
+
+nav a:first-child {
+    border-radius: 10% 0 0 10%;
+}
+
+nav a:last-child {
+    border-radius: 0 10% 10% 0;
+}
+
+nav a:hover {
+    background: rgba(237, 234, 229, 0.466);
+    color: #026670;
+}
+
+/* Main Section */
+
+.main {
+
+    display: flex;
+    align-items: flex-end;
+}
+
+.main .img {
+    /* box-sizing: content-box;
+    want to incl the margin in the 50% of screen, vs img:50%+margin:4%
+    not border-box either though....*/
```

Comment:
There is no way for a margin to be included in the width. `box-sizing: border-box` will include both the border and the padding, as you've noted. The only way to get what I think you want here is with an outer and inner box. The outer one has a `width: 50%; box-sizing: border-box; padding: 2rem;` and the inner has the actual content and background.

Reply:
Since shrinking the img file, I was having the opposite problem of the img being too small. Setting equal widths for the div's containing each half solved it.

Reply:
If you give the image a width of 100% and leave the height unspecified, it will maintain the aspect ratio, but fill up the container.